src/dftbp/dftbplus/initprogram.F90

```diff
@@ -1713,11 +1742,20 @@ subroutine initProgramVariables(this, input, env)
     allocate(this%img2CentCell(this%nAllAtom))
     allocate(this%iCellVec(this%nAllAtom))
 
+    if (this%isHybridXc) then
```

Comment:
Probably a good idea to have a general isSymNeighbourNeeded variable, which is set = isHybridXc for the moment, as this would make it easier to generalise other cases later. (will be needed to fix solvation)

Reply:
I guess this is addressed by your PR for now.